docs/src/modules/components/AppTableOfContents.js

```diff
@@ -67,12 +65,13 @@ const NavItem = styled(Link, {
 
   return [
     {
+      display: 'block',
       fontSize: theme.typography.pxToRem(13),
-      padding: theme.spacing(0, 1, 0, secondary ? 2.5 : '10px'),
-      margin: theme.spacing(0.5, 0, 1, 0),
+      fontWeight: theme.typography.fontWeightMedium,
+      overflowWrap: 'break-word',
```

Comment:
I would be more in favor of

```suggestion
      textOverflow: 'ellipsis',
      overflow: 'hidden',
```

The reason is that most of the time large texts are due to long props names. One of the demo is the data grid premium API you can have a look here: https://deploy-preview-39417--material-ui.netlify.app/experiments/docs/data-grid-premium/



<table>
<tr>
 <td>Wrap
 <td>Elipse
<tr>
 <td><img src='https://github.com/mui/material-ui/assets/45398769/f93f19fe-f12c-4aa0-9754-1137af86789f' />
 <td><img src='https://github.com/mui/material-ui/assets/45398769/56fee42e-e7c6-4308-8bdd-7fcee12de1c8' />
</table>

Reply:
That makes sense!